flink-runtime/src/test/java/org/apache/flink/runtime/rpc/akka/AkkaRpcServiceTest.java

```diff
@@ -135,7 +135,7 @@ public void testGetAddress() {
 	 * @throws ExecutionException
 	 * @throws InterruptedException
 	 */
-	@Test(timeout = 1000)
+	@Test
```

Comment:
@tillrohrmann Is is OK to remove these timeouts? We still catch this with the build timeout stack traces if no progress happens. As an alternative, we can increase the timeout to a higher value.